providers/gce/gce_networks.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gce
+
+import (
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud"
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/meta"
+
+	compute "google.golang.org/api/compute/v1"
+)
+
+func newNetworkMetricContext(request string) *metricContext {
+	return newGenericMetricContext("subnetworks", request, unusedMetricLabel, unusedMetricLabel, computeV1Version)
```

Comment:
Should this be 'networks' and not 'subnetworks'

Reply:
good catch, fixed.